aries-site/src/pages/components/table.mdx

```diff
@@ -161,6 +161,20 @@ In this context, a Table should facilitate the assembly of the data set and make
 accessible in a tool specifically designed for analysis.
 -->
 
+## Guidelines
+
+### Setting the width of the table
+
+- Use the right set of columns, prioritizing frequently used ones. Don't add more columns just because the data is there, as this will clutter the table.
```

Comment:
"right" can be confused with direction

```suggestion
- Use the correct set of columns, prioritizing frequently used ones. Don't add more columns just because the data is there, as this will clutter the table.
```